### PR TITLE
Simple differ for Eco files

### PR DESCRIPTION
**Pull request by ptersilie**
Allows the diffing of Eco files by exporting them into plain-text and then comparing the results.

### CODE REVIEW

File: bin/diffeco [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        *)
+            usage
+            ;;
+    esac
+done
+
+BASEDIR=$(dirname "$0")
+
+if [ -z $file2 ]; then
+    echo "Ecodiff $file1 and $commit"
```
Comment (ltratt) [inline bin/diffeco]: `ecodiff` lower case?

File: bin/diffeco [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    git archive $commit $file1 | tar -xO > /tmp/ecodiff0
```
Comment (ltratt) [inline bin/diffeco]: Let's use `mktemp` for the temporary files so we avoid race conditions.

File: bin/diffeco [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    $BASEDIR/eco -e /tmp/ecodiff0 /tmp/ecodiff2
+    rm /tmp/ecodiff0
+else
+    echo "Ecodiff $file1 and $file2"
```
Comment (ltratt) [inline bin/diffeco]: `ecodiff` lower case.

**Comment by ptersilie** [comment]
Fixed

**Comment by ltratt** [comment]
Looks good. Please squash.

**Comment by ptersilie** [comment]
Squashed!

**Review by ltratt — APPROVED** [review]

**Pull request merged**